src/main/java/com/google/cloud/tools/appengine/cloudsdk/DevAppServer.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.appengine.cloudsdk;
+
+import com.google.cloud.tools.appengine.api.devserver.AppEngineDevServer;
+import com.google.cloud.tools.appengine.cloudsdk.process.ProcessHandler;
+import com.google.common.annotations.VisibleForTesting;
+
+/** Create Dev App Servers. */
+public class DevAppServer {
```

Comment:
A better name could make the distinction between DevAppServer and AppEngineDevServer much clearer.